etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/StageLoad.java

```diff
@@ -76,35 +71,17 @@ private void processStageLoad(String stageDbName, String viewName, String baseDb
 
             Connection rawCon = getHiveJDBCConnection(stageDbName);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "rawCon" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1481)